src/main/java/christmas/view/InputView.java

```diff
@@ -0,0 +1,42 @@
+package christmas.view;
+
+import static christmas.controller.Parser.parseDate;
+import static christmas.controller.Parser.parseMenus;
+
+import camp.nextstep.edu.missionutils.Console;
+import christmas.exception.ErrorMessage;
+import christmas.exception.PromotionException;
+import java.util.List;
+
+public class InputView {
+    private static final String DATE_PATTERN = "[\\d]{1,2}";
+    private static final String MENUS_PATTERN = "(([가-힣]+)-([\\d]{1,2}),)*([가-힣]+)-([\\d]{1,2})";
+
+    public int readVisitDate() {
+        System.out.println("12월 중 식당 예상 방문 날짜는 언제인가요? (숫자만 입력해 주세요!)");
+        String date = Console.readLine();
+        validateDateInput(date);
+        return parseDate(date);
+    }
+
+    private void validateDateInput(String input) {
+        if (input.matches(DATE_PATTERN)) {
+            return;
+        }
+        throw new PromotionException(ErrorMessage.INVALID_DATE_MESSAGE);
+    }
+
+    public List<String> readMenus() {
+        System.out.println("주문하실 메뉴를 메뉴와 개수를 알려 주세요. (e.g. 해산물파스타-2,레드와인-1,초코케이크-1)");
```

Comment:
비록 한번만 쓰일 문장이여도 리펙토링이 편하고 가독성을 위해 상수로 분리하는 것이 좋다고 생각합니다!

```suggestion
private final static String ASK_ORDER_MESSAGE = "주문하실 메뉴를 메뉴와 개수를 알려 주세요. (e.g. 해산물파스타-2,레드와인-1,초코케이크-1)";

        System.out.println(ASK_ORDER_MESSAGE);
```